.pre-commit-config.yaml

```
@@ -19,6 +19,12 @@ repos:
       - id: mixed-line-ending
       - id: trailing-whitespace
 
+  - repo: https://github.com/psf/black
+    rev: "25.1.0"
+    hooks:
+      - id: black
+        args: [--preview]
```

```suggestion
        args: [--preview, --unstable, string_processing]
```

that way we get [the nice long-string-wrap](https://github.com/psf/black/issues/4196).


Also, can you delete on line 34 the "`- id: ruff-format`"? that way we dont have 2 format processing hooks.